### DISCUSSION TITLE
Bigger plot for all in the data prep????

### DISCUSSION THREAD

**Discussion by juhonkang** [sfu-db/dataprep#897] [Q&A]
Hi, I want to make the bigger plots for all the EDA, could anybody has a way?
Please help me, thanks!

**Comment by jinglinpeng** [comment]
Hi @juhonkang , have you tried the `width` and `height` parameter in the `config`? E.g., `plot(df, config = {'width':xxx, 'height':xxx})`

**Comment by juhonkang** [comment]
Yes, I have tried it but do you have any solution to make the proportional way. Like config = {'width':80%, height:'100%'}. Because the feature is different => changing the config pixely every time is a very tedious task.
